config/sync/views.view.user_admin_people.yml

```diff
@@ -458,6 +412,70 @@ display:
           multi_type: separator
           separator: ', '
           field_api_classes: false
+        mail_1:
```

Comment:
I think it's good to have this field on both displays, but as long as you're in there, let's move the "Operations" field/column all the way to the right on the form.